benchmarks/styled-components/src/components/ReactTable.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export default function Table({ table, toPercent }) {
+const ReactTable = function Table({ table, toPercent }) {
```

Comment:
just `export default function ReactTable` is fine.

Reply:
I changed it because there was no visual indication of which table was being rendered on the screen. On line 17 I added a `displayName` property that is accessed later on to display the current component being rendered.